packages/generator/src/generator.ts

```diff
@@ -91,6 +102,10 @@ export abstract class Generator<T extends GeneratorOptions = GeneratorOptions> e
       const additionalFilesToIgnore = this.filesToIgnore()
       return ![...alwaysIgnoreFiles, ...additionalFilesToIgnore].includes(name)
     })
+    const prettierOptions = (await prettier.resolveConfig(this.sourcePath())) || {
+      semi: false,
+      printWidth: 110,
+    }
```

Comment:
I think we should not set default config values here. Instead, if config isn't found, then just fallback to the prettier default by not explicitly setting options.

Make sense?

Reply:
Yes that makes sense, let me change that